### PR TITLE
Practitioner Reference Number and Practitioner description 

### PR DESCRIPTION
**Pull request by bart3k**
Ref: #76

### CODE REVIEW

File: sections/practitioner_reference_numbers.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+  "name": "Provider #",
+  "reference_number": "123123AB",
+  "updated_at": "2013-03-26T14:00:00Z",
+  "business": {
+    "links": {"self": "http://api.cliniko.dev:3000/v1/businesses/1"}
+  },
+  "practitioner": {
+    "links": {"self": "http://api.cliniko.dev:3000/v1/practitioners/1"}
+  },
+  "links": {"self": "https://api.cliniko.com/v1/practitioner_reference_numbers/1"}
```
Comment (bpinto) [inline sections/practitioner_reference_numbers.md]: All links on this file are not nested under practitioner:
https://api.cliniko.com/v1/practitioners/1/practitioner_reference_numbers?page=1

File: sections/practitioner_reference_numbers.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
+Practitioner Reference Numbers
+============
+* [Get Practitioner Reference Numbers](#get-practitioner-reference-numbers "This will return all practitioner reference numbers.")
+* [Get Practitioner Reference Number](#get-practitioner-reference-number "This will return a specified practitioner reference number.")
+
+Get Practitioner Reference Numbers
+----------------
+
+**Resources**
+* ```GET /practitioner_reference_numbers``` get all practitioner reference numbers.
```
Comment (jfriedlaender) [inline sections/practitioner_reference_numbers.md]: We don't want this end point.  Only for a specific practitioner.

File: sections/practitioner_reference_numbers.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+  "total_entries": 1,
+  "links": {"self": "https://api.cliniko.com/v1/practitioner_reference_numbers?page=1"}
+}
+```
+
+Get Practitioner Reference Number
+------------
+
+**Resources**
+* ```GET /practitioner_reference_numbers/:id``` get a specified practitioner reference number.
```
Comment (jfriedlaender) [inline sections/practitioner_reference_numbers.md]: I think this should be....   /practitioners/:practitioner_id/practitioner_reference_numbers/:id

File: sections/practitioners.md [hunk 4]
```diff
@@ -38,6 +39,9 @@ curl https://api.cliniko.com/v1/practitioners \
       "default_appointment_type": {
         "links": {"self": "https://api.cliniko.com/v1/appointment_types/1"}
       },
+      "practitioner_reference_numbers": {
+        "links": {"self": "https://api.cliniko.com/v1/practitioners/1/practitioner_reference_numbers?page=1"}
```
Comment (jfriedlaender) [inline sections/practitioners.md]: Do we specify page by default?

Comment (bpinto) [inline sections/practitioners.md]: We do.

**Comment by jfriedlaender** [comment]
:+1:

**Pull request merged**